<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qdmr [*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <property name="accessibleName">
   <string>Main window</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="accessibleName">
       <string>Tablist</string>
      </property>
      <property name="accessibleDescription">
       <string>There are 6 tabs providing general settings, contact, Rx group, channel zone and scan lists.</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewCodeplug"/>
    <addaction name="actionCodeplugWizard"/>
    <addaction name="actionOpenCodeplug"/>
    <addaction name="actionSaveCodeplug"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionDetectDevice"/>
    <addaction name="actionVerifyCodeplug"/>
    <addaction name="separator"/>
    <addaction name="actionDownload"/>
    <addaction name="actionUpload"/>
    <addaction name="actionUploadCallsignDB"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuDatabases">
    <property name="title">
     <string>Databases</string>
    </property>
    <addaction name="actionRefreshCallsignDB"/>
    <addaction name="actionRefreshTalkgroupDB"/>
    <addaction name="actionRefreshRepeaterDB"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
   <addaction name="menuDatabases"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="accessibleName">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewCodeplug"/>
   <addaction name="actionOpenCodeplug"/>
   <addaction name="actionSaveCodeplug"/>
   <addaction name="separator"/>
   <addaction name="actionDetectDevice"/>
   <addaction name="actionVerifyCodeplug"/>
   <addaction name="actionDownload"/>
   <addaction name="actionUpload"/>
   <addaction name="actionUploadCallsignDB"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionNewCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Creates a new Codeplug.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCodeplugWizard">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dialog.png</normaloff>:/icons/dialog.png</iconset>
   </property>
   <property name="text">
    <string>New with wizard ...</string>
   </property>
   <property name="toolTip">
    <string>Creates a new codeplug with a wizard.</string>
   </property>
  </action>
  <action name="actionOpenCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Imports a codeplug from &amp;quot;conf&amp;quot; files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save ...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Saves the codeplug in a &amp;quot;conf&amp;quot; file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quits the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDetectDevice">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
   <property name="toolTip">
    <string>Detect connected radios.</string>
   </property>
  </action>
  <action name="actionVerifyCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Verifies the current codeplug with connected radios.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Downloads a codeplug from connected radios.</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/upload.png</normaloff>:/icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Uploads the codeplug to the connected radio.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About qdmr</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Read the handbook.</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Shows settings dialog</string>
   </property>
  </action>
  <action name="actionUploadCallsignDB">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/userupload-4x.png</normaloff>:/icons/userupload-4x.png</iconset>
   </property>
   <property name="text">
    <string>Upload Callsign DB</string>
   </property>
   <property name="toolTip">
    <string>Upload call-sign DB to radio.</string>
   </property>
  </action>
  <action name="actionRefreshCallsignDB">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cloud-download-4x.png</normaloff>:/icons/cloud-download-4x.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Callsign DB</string>
   </property>
   <property name="toolTip">
    <string>Refreshes the downloaded callsign DB</string>
   </property>
  </action>
  <action name="actionRefreshTalkgroupDB">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cloud-download-4x.png</normaloff>:/icons/cloud-download-4x.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Talkgroup DB</string>
   </property>
   <property name="toolTip">
    <string>Refreshes the downloaded talkgroup DB</string>
   </property>
  </action>
  <action name="actionRefreshRepeaterDB">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cloud-download-4x.png</normaloff>:/icons/cloud-download-4x.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Repeater DB</string>
   </property>
   <property name="toolTip">
    <string>Refreshes the downloaded repeater DB</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
